mysql-test/suite/rpl/t/rpl_repl_slave_set_gtid_next.test

```diff
@@ -0,0 +1,81 @@
+source include/assert_gtid_mode_on.inc;
```

Comment:
This test does not set up replication - may be it should be put into `binlog_gtid` suite.